src/js/WinJS/Utilities/_TelemetryImpl.js

```diff
@@ -10,7 +11,7 @@ define([
 
     /// Make sure to use these hard-coded strings
     var MicrosoftGroup = "4f50731a-89cf-4782-b3e0-dce8c90476ba";
-    var MicrosoftKeyword = "0x2000000000000";
+    var MicrosoftKeyword = 0x200000000000;
```

Comment:
It looks like this number lost a 0. Was that intentional? How many zeroes should it have?